Content.Server/GameObjects/Components/Projectiles/ProjectileComponent.cs

```diff
@@ -36,6 +36,8 @@ public class ProjectileComponent : SharedProjectileComponent, ICollideBehavior
         private string _soundHit;
         private string _soundHitSpecies;
 
+        private bool _damagedEntity;
```

Comment:
Looks like you mixed this with your other PR.

Reply:
Oops